### DISCUSSION TITLE
Tag options - To restore

### DISCUSSION THREAD

**Discussion by robsori** [frangoteam/FUXA#1300] [Q&A]
Hi. Does anyone know the function of the "To restore" setting in the "Tag options" window? Thank you.

![Tag options](https://github.com/user-attachments/assets/6130a87b-aae6-4a35-8ee5-c67045e8f80d)

**Comment by unocelli** [comment]
Hi, if you enable restore, fuxa will save the last value and next new start of fuxa the value will be set (restored), is typically used for tags used as parameters.
